### PR TITLE
LOG-2381: Vector: Add missing Normalize container_logs transform for infra logs

### PR DESCRIPTION
**Pull request by vimalk78**
### Description
Added missing Normalize container_logs transform for infra logs

/cc 
/assign 

### Links

- JIRA: https://issues.redhat.com/browse/LOG-2381

### CODE REVIEW

File: internal/generator/vector/conf.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func Conf(clspec *logging.ClusterLoggingSpec, secrets map[string]*corev1.Secret,
 func NormalizeLogs(spec *logging.ClusterLogForwarderSpec, op generator.Options) []generator.Element {
 	types := generator.GatherSources(spec, op)
 	var el []generator.Element = make([]generator.Element, 0)
-	if types.Has(logging.InputNameApplication) {
+	if types.Has(logging.InputNameApplication) || types.Has(logging.InputNameInfrastructure) {
```
Comment (jcantrill) [inline internal/generator/vector/conf.go:38]: Does it matter if journal logs come through this transformation?

Comment (vimalk78) [reply internal/generator/vector/conf.go:38]: journal logs come from another source `journald`

**Pull request merged**